ocs_ci/ocs/tests/test_fiojob.py

```diff
@@ -0,0 +1,472 @@
+# -*- coding: utf8 -*-
+
+from unittest.mock import patch, Mock
+import logging
+import textwrap
+
+import pytest
+import yaml
+import yaml.parser
+
+from ocs_ci.ocs import fiojob
+
+
+@pytest.fixture
+def fio_json_output():
+    """
+    Example of fio --output-format=json output. Based on actual test run.
+    """
+    fio_out = textwrap.dedent("""
```

Comment:
suggesting maybe to save this is a template file

Reply:
This is unit test of ocs-ci code, not ocs-ci test. If there were more unit tests which works with fio output, it could make sense to save this into dedicated file, but there are no such tests. That said, we may do that later.

Reply:
I'm going to push this out into a separate file next to this test file.

Reply:
Fixed in 40f78658bad73e180b2b18520e5cc9a4bec2c382